tests/unittests/config/test_apt_source_v3.py

```diff
@@ -1540,3 +1541,57 @@ def test_disable_deb822_suites_disables_proper_suites(
         assert expected == cc_apt_configure.disable_suites_deb822(
             disabled_suites, src, "mantic"
         )
+
+
+APT_CONFIG_DUMP = """
+APT "";
+Dir "/";
+Dir::Etc "etc/myapt";
+Dir::Etc::sourcelist "sources.my.list";
+Dir::Etc::sourceparts "sources.my.list.d";
+Dir::Etc::main "apt.conf";
+"""
+
+
+class TestGetAptCfg:
+    @skipIfAptPkg()
```

Comment:
Because we aren't defining apt_pkg in our requirements.txt or test-requirement.txt file, these tests will always run by default in our development environment. During pkg build and integration tests, python3-apt will almost always be installed due to other build dependencies or base ubuntu images, so these tests will be skipped as they only rely on calling apt-config dump if apt_pkg is not present. So, we get coverage with and without this dependency in various parts of our testing.